UserInterface/MainWindow.cpp

```diff
@@ -1765,24 +1767,20 @@ void OpenInfraPlatform::UserInterface::MainWindow::on_actionShow_Help_triggered(
 }
 
 void OpenInfraPlatform::UserInterface::MainWindow::on_actionShow_Log_File_triggered() {
+#ifndef _DEBUG
 	wchar_t* localAppData = 0;
 	SHGetKnownFolderPath(FOLDERID_LocalAppData, 0, NULL, &localAppData);
 
 	std::wstringstream ss;
-	ss << localAppData << L"/OpenInfraPlatform/";
+	ss << localAppData << _T("/OpenInfraPlatform/") << _T("log.txt");
 
 	CoTaskMemFree(static_cast<void*>(localAppData));
-	// setup converter
-	std::wstring_convert<convert_type, wchar_t> converter;
-
-	// use converter (.to_bytes: wstr->str, .from_bytes: str->wstr)
-	std::wstring filename = L"log.txt";
 
-#ifndef _DEBUG
-	filename = ss.str().append(L"log.txt");
+	ShellExecute(0, 0, ss.str().c_str(), 0, 0, SW_SHOW);
+#else
+	ShellExecute(0, 0, _T("log.txt"), 0, 0, SW_SHOW);
 #endif
 
-	ShellExecute(0, 0, filename.c_str(), 0, 0, SW_SHOW);
 }

```

Comment:
We aren't writing to log file anymore; we can remove the function alltogether.

Reply:
Done